resources/js/src/app/components/itemList/filter/ItemCategoryDropdown.vue

```diff
@@ -113,8 +116,61 @@ export default defineComponent({
 
       return '';
     };
+
+    const isSelected = computed((): boolean => {
+      if (typeof props.currentCategory !== 'undefined' && isParentCategorySelected()) {
+        return false;
+      }
+
+      return typeof UrlBuilder.getSelectedFilters().find(element => element.id === props.facet.id) !== 'undefined';
+    });
+
+    const isInCategoryPage = computed((): boolean => {
+      return typeof props.currentCategory !== 'undefined';
+    });
+
+    const getCategories = computed((): FacetValue[] | undefined  => {
```

Comment:
As this is a `computed`, I would not give it a getter name. Just `categories` would be enough.